argilla-server/src/argilla_server/api/schemas/v1/datasets.py

```diff
@@ -87,15 +87,15 @@ class DatasetProgress(BaseModel):
 
 
 class RecordResponseDistribution(BaseModel):
-    submitted: Optional[int]
-    discarded: Optional[int]
-    draft: Optional[int]
+    submitted: int = 0
+    discarded: int = 0
+    draft: int = 0
 
 
 class UserProgress(BaseModel):
     username: str
-    completed: RecordResponseDistribution
-    pending: RecordResponseDistribution
+    completed: RecordResponseDistribution = RecordResponseDistribution()
+    pending: RecordResponseDistribution = RecordResponseDistribution()
```

Comment:
Is this change really needed? You already added default values to `RecordResponseDistribution`.